bigtable/tests/unit/test_table.py

```diff
@@ -563,6 +563,91 @@ def test_read_row_complete(self):
         column.append(Cell.from_pb(chunk))
         self._read_row_helper(chunks, expected_result, app_profile_id)
 
+    def test_read_row_retry(self):
```

Comment:
This test should be something like `test_read_row_custom_retry`, and should only test that the `retry_read_row` is passed correctly from `read_row` to `read_rows`.  There's a lot of other stuff in this method that is "orthogonal" to the specific thing you're trying to test, and therefore should be removed; streamlining the test will allow future reviewers to quickly understand the intent of the test.